packages/commonwealth/client/scripts/views/pages/MyCommunityStake/NoTransactionHistory/NoTransactionHistory.tsx

```diff
@@ -0,0 +1,38 @@
+import { useCommonNavigate } from 'client/scripts/navigation/helpers';
+import { CWText } from 'client/scripts/views/components/component_kit/cw_text';
+import { CWButton } from 'client/scripts/views/components/component_kit/new_designs/cw_button';
+import React from 'react';
+import './NoTransactionHistory.scss';
+
+const NoTransactionHistory = () => {
+  const navigate = useCommonNavigate();
+
+  return (
+    <section className="NoTransactionHistory">
+      <div className="container">
+        <img
+          src="/static/img/noTransactionHistory.svg"
+          alt="no transaction history icon"
+        />
+        <div className="labels">
+          <CWText type="h4" fontWeight="semiBold">
+            You have not purchased stake in any communities
+          </CWText>
+          <CWText type="b1">
+            <span>
+              Purchasing community stake gives you more upvote power within your
+              communities. <a href="#">Learn more</a>
```

Comment:
do we know where this href should take a user?

Reply:
asked from design/product

Reply:
done